src/styles/variables.scss

```
@@ -0,0 +1,22 @@
+$card-width: 198px; // ширина карти з урахуванням бордюру буде 200px
```

```suggestion
$card-width: 200px;
```

And add `box-sizing: border-box;`